deployment/network-operator/charts/node-feature-discovery/values.yaml

```diff
@@ -1,9 +1,10 @@
 image:
-  repository: registry.k8s.io/nfd/node-feature-discovery
+  repository: ghcr.io/kubernetes-sigs/node-feature-discovery
   # This should be set to 'IfNotPresent' for released version
   pullPolicy: IfNotPresent
   # tag, if defined will use the given image tag, else Chart.AppVersion will be used
-  # tag
+  tag: v0.15.3
```

Comment:
This file is not needed since you're controlling these fields via the network-operator chart now